src/pool/rpc.h

```diff
@@ -358,14 +358,15 @@ CRT_RPC_DECLARE(pool_tgt_query, DAOS_ISEQ_POOL_TGT_QUERY,
 CRT_RPC_DECLARE(pool_tgt_update_map, DAOS_ISEQ_POOL_TGT_UPDATE_MAP,
 		DAOS_OSEQ_POOL_TGT_UPDATE_MAP)
 
-#define DAOS_ISEQ_POOL_GET_ACL	/* input fields */		 \
-	((struct pool_op_in)	(pgi_op)		CRT_VAR)
+#define DAOS_ISEQ_POOL_PROP_GET	/* input fields */		 \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
These false positives come out of every CaRT change. Please ignore.